email-sending-concepts/one-click-unsubscribe.md

```diff
@@ -64,49 +64,57 @@ head:
+One-Click Unsubscribe provides email recipients with a standardized, **immediate method** to remove themselves from email lists or marketing communications. It appears as a special link or button near the sender information in modern [email clients](/email-sending-concepts/smtp.md) (like Gmail's "Unsubscribe" button). When clicked, it immediately processes the unsubscribe request **without requiring additional steps** from the user.
 
 ## How One-Click Unsubscribe Works
 
-The One-Click Unsubscribe mechanism is primarily implemented through standardized [email headers](/email-sending-concepts/email-headers) that work behind the scenes to facilitate the unsubscribe process. Email senders include special headers in their messages that email clients recognize and can present as unsubscribe options to users.
-
-The implementation centers around the **`List-Unsubscribe`** and **`List-Unsubscribe-Post`** headers, which provide recipient email clients with the necessary information to process unsubscribe requests automatically.
+The One-Click Unsubscribe mechanism is primarily implemented through standardized [email headers](/email-sending-concepts/email-headers) that facilitate the process:
 
 | Header Type | Purpose | Example |
 |-------------|---------|---------|
 | **List-Unsubscribe (HTTP)** | Provides a URL endpoint for unsubscribe requests | `<https://example.com/unsubscribe?token=abc123>` |
 | **List-Unsubscribe (Mailto)** | Provides an email address for unsubscribe requests | `<mailto:unsubscribe@example.com?subject=unsubscribe>` |
 | **List-Unsubscribe-Post** | Enables true one-click functionality with secure POST requests | `List-Unsubscribe=One-Click` |
 
-Modern email clients like Gmail and Apple Mail scan these headers and display user-friendly unsubscribe options, typically as buttons or links near the sender information. When users click these options, the client handles the technical complexity of making the proper HTTP request or sending the unsubscribe email automatically.
+Modern email clients like Gmail and Apple Mail scan these headers and display user-friendly unsubscribe options (e.g., buttons or links near the sender information). When users click these options, the client handles the technical process of sending the proper HTTP request or unsubscribe email automatically.
+
+> **Note:** When users unsubscribe via one-click, they are only unsubscribed from the specific list through which they received the email. To unsubscribe from all lists or categories, users should visit the subscription preferences page provided through the unsubscribe link, where they can opt out of all communications if desired.
```

Comment:
I would say "When users unsubscribe via the one-click unsubscribe method, ...

Reply:
well... you can opt out... if they implemented it in that way. (And they should)